src/BuildingRegistry.Api.BackOffice.Abstractions/BuildingUnit/Responses/PlanBuildingUnitResponse.cs

```diff
@@ -1,6 +1,7 @@
 namespace BuildingRegistry.Api.BackOffice.Abstractions.BuildingUnit.Responses
 {
     public record PlanBuildingUnitResponse(
+        int BuildingPersistentLocalId,
```

Comment:
I think this field is not needed to return, the controller does not use this field.